### PR TITLE
save -> saveValue

### PR DESCRIPTION
**Pull request by e111077**
`app-storage` 2.0.0 changed overridable `save` to `saveValue`. I don't know why we never updated this element. bower conflicts in tests seem to be because of platinum-sw. see #50

### CODE REVIEW

File: demo/note-app.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
     <na-toolbar></na-toolbar>
 
-    <app-pouchdb-confict-resolution
```
Comment (valdrinkoshi) [inline demo/note-app.html]: 🎊 😆

File: app-pouchdb-document.html [hunk 2]
```diff
@@ -108,7 +108,7 @@
       },
 
       /** @override */
-      save: function() {
+      saveValue: function() {
```
Comment (valdrinkoshi) [inline app-pouchdb-document.html]: we should also override `save` in case the user installs the 1.x variant, right?

Comment (e111077) [inline app-pouchdb-document.html]: Acutally, app-storage v1.0.0 using Polymer 1 uses `saveValue`. we changed this when we went from beta -> 1.0.0 so we can keep it like this

Comment (valdrinkoshi) [inline app-pouchdb-document.html]: The 1.x variant still points to v0.10.0 
https://github.com/PolymerElements/app-pouchdb/blob/master/bower.json#L48

Comment (e111077) [inline app-pouchdb-document.html]: Ah, that should definitely be fixed to point to 1.0.0

Comment (e111077) [inline app-pouchdb-document.html]: fixed in this PR

**Comment by e111077** [comment]
Tests fixed; this PR still relies on PolymerLabs/note-app-elements#13 or else the demo will be broken

**Comment by valdrinkoshi** [comment]
LGTM 👍

**Pull request merged**